generators/newwebapp/index.js

```diff
@@ -178,7 +178,7 @@ module.exports = class extends Generator {
                     }
                 },
                 appOptions: {
-                    loadReuseLibs: false
+                    loadReuseLibs: this.options.oneTimeConfig.platform === "SAP Launchpad service"
```

Comment:
personally I would prefer a constant since this String is used in 3 places in this file, would you mind to introduce a constant here @nicogeburek ?